prep-css-syntax/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <style>
+  .h1 {
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
Try leaving out the `.` here are just use `h1`. Your `font-family` still appears to be Times New Roman.